support/modules-artifact-gen/single-file-artifact-gen

```diff
@@ -155,8 +155,14 @@ echo "$dest_dir" > $dest_dir_file
 # Create single_file file in plain text
 echo "$filename" > $filename_file
 
+set +e;
+STAT_HAS_f=0;
+stat -f %A "${file}" >/dev/null 2>&1;
+[ $? -eq 0 ] && STAT_HAS_f=1;
+set -e;
```

Comment:
I suggest doing all this into a two liner like:
```
STAT_HAS_f=1
stat -f %A "${file}" >/dev/null 2>&1 || STAT_HAS_f=0
```

Reply:
> I suggest doing all this into a two liner like:
> 
> ```
> STAT_HAS_f=1
> stat -f %A "${file}" >/dev/null 2>&1 || STAT_HAS_f=0
> ```

thanks. [modified](https://github.com/mendersoftware/mender/pull/428/commits/2ca77cc03487655e5f25525df2e1fb1518fc3e41) as requested